### ISSUE TITLE
Specifying CDK bootstrap stack name when bootstrapping SST

### ISSUE THREAD

**Issue by jmnavarr** [serverless-stack/sst#3029]
We're trying to get an SST bootstrap going on top of an existing CDK bootstrap. 

For our CDK bootstraps, we're using qualifiers and custom stack names to keep separate bootstraps from separate teams nice and tidy. For bootstrapping SST on top of any CDK bootstraps, we're using sst.config.ts to set info on pre-existing CDK bootstraps.

However, even though we're setting cdk.toolkitStackName to specify a non-default-named CDK bootstrap stack, SST still seems to be looking for the default CDKToolkit stack name during the SST bootstrap process, failing to find it because it doesn't exist, and throwing an error that the CDK deploy role can't perform cloudformation:DescribeStacks on the CDKToolkit stack. But there is no CDKToolkit stack in the account, and we're telling SST about the CDK cfn stack that IS there, and it seems to be ignoring that when running the SST bootstrap.

I have found that the cdk.toolkitStackName property is not used for the SST bootstrap process.

The bootstrap.stackName property is just the name of the SST bootstrap stack.

In summary: there is no configuration that we can specify in that config file that will allow us to state which CDK bootstrap stack we want to use for the SST bootstrap process.

**Comment by jmnavarr** [comment]
I think this PR will fix that issue: https://github.com/serverless-stack/sst/pull/3030/files

**Comment by berenddeboer** [comment]
Very handy to have.

**Comment by fwang** [comment]
Thanks @jmnavarr! Released in 2.17.2
